sp800-63c/sec5_strength.md

```diff
@@ -70,11 +70,11 @@ All assertions SHOULD use audience restriction techniques to allow an RP to reco
 
 #### 5.2.5. Pairwise Pseudonymous Identifiers
 
-In some circumstances, it is desirable to prevent the subscriber's account at the CSP from being linked through one or more RPs through use of a common identifier. Pairwise pseudonymous subject identifiers SHALL be used within the assertions generated by the CSP for the RP, and the CSP SHALL generate a different identifier for each RP.
+In some circumstances, it is desirable to prevent the subscriber's account at the CSP from being linked through one or more RPs through use of a common identifier. Pairwise pseudonymous subject identifiers SHALL be used within the assertions generated by the CSP for the RP, and the CSP SHALL generate a different identifier for each RP or integration with an RP, as appropriate.
 
-When unique pseudonymous identifiers are used with RPs along side of identity attribute bundles, it may still be possible for multiple colluding RPs to fully identify and correlate a subscriber across systems using these attributes. Even though a different pseudonymous identifier was used at each RP, the RPs are still able to compare other identifying attributes such as names, email addresses, physical addresses, and phone numbers, among other common attributes. In such cases, the privacy risk is similar to the subscriber not having used a pseudonymous identifier at all.
+When unique pseudonymous identifiers are used with RPs along side of identity attribute bundles, it may still be possible for multiple colluding RPs to fully identify and correlate a subscriber across systems using these attributes. Privacy policies, therefore, may prohibit such correlation, but pairwise pseudonymous identifiers can increase effectiveness of these policies by increasing the administrative effort in managing the attribute correlation. 
 
-Note that in a brokered federation, a CSP cannot issue a pseudonymous identifier for each RP because it is unaware which RP is being accessed. In this case the broker, as a proxy CSP, may wish to generate pseudonymous identifiers to prevent privacy-violating collusion among RPs. Depending on the protocol, the broker may often need to map such identifiers back to the associated identifiers from upstream CSPs in order to allow the identity protocol to function.
+Note that in a distributed federation model, the distributing party that is acting as a proxy CSP may, depending on the protocol, need to map the pairwise pseudonymous identifiers back to the associated identifiers from upstream CSPs in order to allow the identity protocol to function.
```

Comment:
The 'broker' performs roles similar to gateways, address translators or
proxy servers in other protocol types.

the important functions seem to be: identifier mapping/translation; 'proxy
server' functions to make 2-connection connections to isolate the sender &
receiver; 'gateway' functions to do protocol translation; and authorization
management to control endpoint transmission permissions.

So, functionally, is it:
- a pseudonymous identifier translator if thinking in 'identity network'
terms
- a credential gateway if thinking in ICAM terms (maybe)
- a pseudonymizing assertion proxy server if in federated assertion
authority terms - but maybe not :-)




*Andrew Hughes *CISM CISSP
Independent Consultant
*In Turn Information Management Consulting*

o  +1 650.209.7542
m +1 250.888.9474
1249 Palmer Road,
Victoria, BC V8P 2H8
AndrewHughes3000@gmail.com
ca.linkedin.com/pub/andrew-hughes/a/58/682/
*Identity Management | IT Governance | Information Security *

On Mon, Aug 29, 2016 at 8:28 AM, pgrassi-nist <notifications@github.com>
wrote:

> In sp800-63c/sec5_strength.md
> <https://github.com/usnistgov/800-63-3/pull/289#discussion_r76627579>:
>
> >
> > -Note that in a brokered federation, a CSP cannot issue a pseudonymous identifier for each RP because it is unaware which RP is being accessed. In this case the broker, as a proxy CSP, may wish to generate pseudonymous identifiers to prevent privacy-violating collusion among RPs. Depending on the protocol, the broker may often need to map such identifiers back to the associated identifiers from upstream CSPs in order to allow the identity protocol to function.
> > +Note that in a distributed federation model, the distributing party that is acting as a proxy CSP may, depending on the protocol, need to map the pairwise pseudonymous identifiers back to the associated identifiers from upstream CSPs in order to allow the identity protocol to function.
>
> We are attempting to get rid of the term broker. So while not a
> distributed system per se, it's better. De-centralized?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/usnistgov/800-63-3/pull/289/files/1e15acc3062e1ef38f0133d2bcb7deb0891ad99a#r76627579>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ASYM_K36R23DNGF5l0mlp5gWGOPe6XFcks5qkvqJgaJpZM4JrKrQ>
> .
>